packages/connector-github/README.md

```diff
@@ -1,32 +1,85 @@
+      - [创建并配置 OAuth 应用程序](#创建并配置-oauth-应用程序)
+      - [管理 OAuth 应用程序](#管理-oauth-应用程序)
+      - [创建并配置 GitHub 应用程序](#创建并配置-github-应用程序)
+      - [管理 GitHub 应用程序](#管理-github-应用程序)
+  - [编写连接器的 JSON](#编写连接器的-json)
+    - [配置类型](#配置类型)
+  - [测试 GitHub 连接器](#测试-github-连接器)
+  - [参考](#参考)
+
+## Get started
+
+The GitHub connector enables end-users to have access to your application using their own GitHub accounts via GitHub OAuth 2.0 authentication protocol.
+
+### Sign in with GitHub account
 
 Go to the [GitHub website](https://github.com/) and sign in with your GitHub account. You may register a new account if you don't have one.
 
-## Create and configure GitHub OAuth app
+### Create, config and manage apps
 
-Follow the [Creating an OAuth App](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app) guide, and register a new application.
+In GitHub, you can create two different kinds of [apps](https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps) (_GitHub App_ and _OAuth App_) as prerequisites before enabling end-users' sign-in.
 
-Name your new OAuth application in **Application name** and fill up **Homepage URL** of the app.
-You can leave **Application description** field blank and customize **Authorization callback URL** as `${your_logto_origin}/callback/github-universal`.
+You may check out [difference between them](https://docs.github.com/en/developers/apps/getting-started-with-apps/differences-between-github-apps-and-oauth-apps) for more details.
+
+#### Create and configure OAuth app
 
-We suggest not to check the box before **Enable Device Flow**, or users who sign in with GitHub on mobile devices must confirm the initial sign-in action in the GitHub app. Many GitHub users do not install the GitHub mobile app on their phones, which could block the sign-in flow. Please ignore our suggestion if you are expecting end-users to confirm their sign-in flow.
+Follow the [creating an OAuth App](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app) guide, and register a new application.
 
-You can also enable end-users to sign in with GitHub accounts by [creating a GitHub app](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app).
+Name your new OAuth application in **Application name** and fill up **Homepage URL** of the app.
+You can leave **Application description** field blank and customize **Authorization callback URL** as `${your_logto_origin}/callback/github-universal`.
 
-## Managing GitHub OAuth apps
+#### Managing OAuth apps
 
 Go to the [OAuth Apps page](https://github.com/settings/developers) and you can add, edit or delete existing OAuth apps.
 You can also find `Client ID` and generate `Client secrets` in OAuth app detail pages.
 
-As for managing GitHub Apps we mentioned in the last section, go to the [GitHub Apps page](https://github.com/settings/apps).
+#### Create and configure GitHub app
```

Comment:
Didn't follow this. pending

Reply:
Discussed offline, should only leave OAuth Apps in the README guide. GitHub Apps is beyond this scope.